tutorial/Flyer_Basics.ipynb

```diff
@@ -0,0 +1,985 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Flyer Basics\n",
+    "\n",
+    "We want to understand how to build a *Flyer* in \"BlueSky\" to support various types of fly scans and remote data loggers.  The data about Flyers is spread about the standard documentation.  We need some clarity and a few examples that build complexity incrementally.\n",
+    "\n",
+    "The basic notion of a Flyer is that it directs an *external controller* (we'll call it the *controller* here) to perform some data colelction process.  Usually, a *controller* is used to collect data at rates beyond the capabilities of BlueSky plans and the RunEngine.  Examples could be waveforms from a storage oscilloscope or a continuous motion scan of a diffractometer.\n",
+    "\n",
+    "This notebook will show the basic requirements for a Flyer and build a simple working example you can use a template for your own work.\n",
+    "\n",
+    "\n",
+    "## Python imports and definitions <a class=\"anchor\" id=\"imports\" />\n",
+    "\n",
+    "Here are the full set of packages to imported.  The first block are Python standard packages, then come the ophyd, BluSky, and databroker packages.  Just the parts we plan on using here.  Since this is also a tutorial, we will not rename imports or use other such shortcuts in the documentation (the online code has some shortcuts).\n",
+    "\n",
+    "* Create a logger instance in case we want to investigate internal details as our code runs.\n",
+    "* Create an instance of the BlueSky RunEngine.\n",
+    "* Create an instance of the databroker using the `mongodb_config.yml` file on the local machine\n",
+    "* Arrange for the databroker to receive all events from the RunEngine\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import logging\n",
+    "import threading\n",
+    "import time\n",
+    "\n",
+    "import ophyd\n",
+    "import bluesky\n",
+    "import bluesky.plans\n",
+    "import databroker\n",
+    "\n",
+    "logger = logging.getLogger()\n",
+    "RE = bluesky.RunEngine({})\n",
```

Comment:
Thanks for the observation!
Personally, I would vote against doing this anymore in the tutorials:
1. It teaches users to use `%run` which I don't really think we should encourage too much
2. It's just a few lines of code and it allows the user to see them without having to open an extra file.
What do you think?

I agree with you that we should decide how to merge both files. I'd vote more to keep them separate with one being "how do I use this with bluesky" and the other as "how to I make a flyer". What do you think?

Reply:
About the use of `%run startup.py`, I'm receiving mixed messages here.  

So here's what I think:
There is a strong consistency motive for all tutorials in this suite to use the same startup.  Yet `startup.py` has grown quite complex and does not cover all of my imports, just the bluesky/ophyd/databroker ones.  As a pedagogical motive, it is better to be explicit *for the student* rather than *convenient* for the editor.

Reply:
Either way is fine to me, however the extra imports (which are not covered by `startup.py`) can be added as well explicitly (unless the sequence is important) before or after `%run startup.py`. Good point about the pedagogical prospective!

Reply:
I'd like to keep the simpler approach.  Switching to use startup.py would go against this assertion in the tutorial "Since this is also a tutorial, we will not rename imports or use other such shortcuts in the documentation".  It uses an import only if it is necessary for the tutorial.